doc/api/process.md

```diff
@@ -706,14 +706,15 @@ console.log(process.env.test);
 // => 1
 ```
 
-## process.emitWarning(warning[, name][, ctor])
+## process.emitWarning(warning[, name[, code]][, ctor])
 <!-- YAML
 added: v6.0.0
 -->
 
 * `warning` {String | Error} The warning to emit.
 * `name` {String} When `warning` is a String, `name` is the name to use
   for the warning. Default: `Warning`.
+* `code` {String} A unique identifier for the warning.
```

Comment:
> Name may be better as "type"?

Remember that `.name` is inherited from the `Error` classes – I don’t think we can change that.

I’m also not worried that much about the naming, a bit of clarification might be all that’s needed